lib/core-classes/AssetService.ts

```diff
@@ -0,0 +1,57 @@
+import omit from 'lodash/omit';
+import { v4 as uuidv4 } from 'uuid';
+
+import { PluginContext, EmbeddedSDK, Plugin, JSONObject } from 'kuzzle';
+
+import { DeviceManagerConfig } from '../DeviceManagerPlugin';
+import { mRequest, mResponse, writeToDatabase } from '../utils/writeMany';
+
+export class AssetService {
+  private config: DeviceManagerConfig;
+  private context: PluginContext;
+
+  get sdk(): EmbeddedSDK {
+    return this.context.accessors.sdk;
+  }
+
+  constructor(plugin: Plugin) {
+    this.config = plugin.config as any;
+    this.context = plugin.context;
+  }
+
+  async importAssets(
+    index: string,
+    assets: JSONObject,
+    { strict, options }: { strict?: boolean; options?: JSONObject }
+  ) {
+    const results = {
+      errors: [],
+      successes: [],
+    };
+
+    const assetDocuments = assets.map((device: JSONObject) => ({
+      _id: device._id || uuidv4(),
```

Comment:
Assets have a default ID of this structure `type-model-reference` rather than a non-human readable uuid